pandas/core/indexes/multi.py

```diff
@@ -1035,11 +1035,13 @@ def is_lexsorted(self):
         """
         return self.lexsort_depth == self.nlevels
 
-    def is_lexsorted_for_tuple(self, tup):
+    def _true_slices_indices(self, tup):
```

Comment:
(or move the error message to the method if you prefer to keep the name and signature)